BUILD

```diff
@@ -6,7 +11,7 @@ exports_files([
 ])
 
 # These two definitions exist to help reduce Envoy upstream core code depending on extensions.
-# To avoid visibility problems, one can replace the package lists below with "//visibility:public"
+# To avoid visibility problems, one can extend ADDITIONAL_VISIBILITY in source/extensions/extensions_build_config.bzl
```

Comment:
re: how to override is documented in https://github.com/envoyproxy/envoy/tree/master/bazel#disabling-extensions, can you update there too?